lib/plugins/create/templates/azure-nodejs/.gitignore

```diff
@@ -0,0 +1,10 @@
+# this is not the file that will be created with the template
```

Comment:
+1

Initial we had a `.gitignore` file in the templates dir, but we had to change this due to the reasons @medikoo mentioned above. The pattern we've established is that we simply remove the `.` in front of the file and the Serverless Framework will automatically rename this file once the template is created.

Reply:
This file is for template writer/dev to test their template changes.  It doesn’t matter if it doesn’t show up in the published version.  

I added it so that I don’t accidentally commit unwanted changes when I’m updating/testing the temple

Reply:
is it ok to put the .gitignore in `lib/plugins/create/templates/`?

Reply:
We may probably add it to `lib/plugins/create/templates` still then the only value that makes sense there is `.serverless`. As `package-lock.json` and `node_modules` are covered by main `.gitignore`, and I don't think it should have Azure specific stuff (there's always some risk it'll collide with files we may want to include in other templates).

So I'm not really sure it's worthwhile.